pkgs/by-name/gr/grafana-to-ntfy/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
+  ...
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "grafana-to-ntfy";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "kittyandrew";
+    repo = pname;
```

The package name doesn't really have anything to do with the repository name.

```suggestion
    repo = "grafana-to-ntfy";
```